<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="dataSourceStorageLocal" created-in="PY-242.23339.19">
    <data-source name="Django default" uuid="514ac27f-7112-49aa-8e4b-6471cc1e2779">
      <database-info product="" version="" jdbc-version="" driver-name="" driver-version="" dbms="SQLITE" />
      <auth-provider>no-auth</auth-provider>
      <schema-mapping>
        <introspection-scope>
          <node kind="schema" negative="1" />
        </introspection-scope>
      </schema-mapping>
    </data-source>
    <data-source name="identifier.sqlite" uuid="9532a67c-7f1a-4796-8663-459a2e6dd62a">
      <database-info product="SQLite" version="3.45.1" jdbc-version="4.2" driver-name="SQLite JDBC" driver-version="3.45.1.0" dbms="SQLITE" exact-version="3.45.1" exact-driver-version="3.45">
        <identifier-quote-string>&quot;</identifier-quote-string>
      </database-info>
      <case-sensitivity plain-identifiers="mixed" quoted-identifiers="mixed" />
      <secret-storage>master_key</secret-storage>
      <auth-provider>no-auth</auth-provider>
      <schema-mapping>
        <introspection-scope>
          <node kind="schema" qname="@" />
        </introspection-scope>
      </schema-mapping>
    </data-source>
  </component>
</project>